File: src/main/java/cz/craftmania/logger/listeners/ChatLoggerListener.java
package cz.craftmania.logger.listeners;

import cz.craftmania.logger.Main;
import org.bukkit.event.EventHandler;
import org.bukkit.event.EventPriority;
import org.bukkit.event.Listener;
import org.bukkit.event.player.AsyncPlayerChatEvent;

public class ChatLoggerListener implements Listener {

    @EventHandler(ignoreCancelled = true, priority = EventPriority.HIGHEST)
    public void onPlayerChat(final AsyncPlayerChatEvent event){
        Main.getInstance().getSQL().createDataLog(event.getPlayer(), "chat", "MESSAGE", event.getMessage(), System.currentTimeMillis());
    }
}

File: src/main/java/cz/craftmania/logger/listeners/internal/ChatLoggerListener.kt
package cz.craftmania.logger.listeners.internal

import cz.craftmania.logger.Main
import cz.craftmania.logger.objects.LogType
import org.bukkit.event.EventHandler
import org.bukkit.event.EventPriority
import org.bukkit.event.Listener
import org.bukkit.event.player.AsyncPlayerChatEvent

class ChatLoggerListener : Listener {

    @EventHandler(ignoreCancelled = true, priority = EventPriority.HIGHEST)
    fun onPlayerChat(event: AsyncPlayerChatEvent) {
        Main.instance!!.sQL!!.createDataLog(event.player, LogType.CHAT, "MESSAGE", event.message, System.currentTimeMillis())
    }
}